src/functions.php

```diff
@@ -380,17 +380,17 @@ function copy_to_stream(
         return;
     }
 
-    $bytes = 0;
-    while (!$source->eof()) {
-        $buf = $source->read($maxLen - $bytes);
-        if (!($len = strlen($buf))) {
+    $bufferSize = 8192;
```

Comment:
Yeah that's true. I am going to reduce the other buffer size to 8k as well. This is what I gather is the default buffer size used in stream operations: http://php.net/manual/en/function.stream-set-write-buffer.php.

I'm going to reduce the buffer size and not raise it because having an overly large buffer size is counter-intuitive to streaming from one stream to another because it requires a large amount of data in memory.